benchmarks/mpas_ocean.py

```diff
@@ -164,3 +164,29 @@ def time_nearest_neighbor_remapping(self):
 
     def time_inverse_distance_weighted_remapping(self):
         self.uxds_480["bottomDepth"].remap.inverse_distance_weighted(self.uxds_120.uxgrid)
+
+from uxarray.grid.coordinates import _construct_face_centerpoints, _construct_face_centroids
```

Comment:
I believe that this import statement is causing the `mpas_ocean` benchmarks to crash for previous runs.


This may also explain the issues you were having locally. I'd put this line inside of the `ConstructFaceLatLon` and it should fix it

![image](https://github.com/user-attachments/assets/4d96ba50-237a-4203-8b3e-d2c911a71ffe)

Reply:
Did you mean to resolve this?

Reply:
> Did you mean to resolve this?

benchmarks seems to run, but profiler doesn't find it still:
```

asv run --bench ConstructFaceLatLon.time_welzl --python=same

Couldn't load asv.plugins._mamba_helpers because
No module named 'libmambapy'
· Discovering benchmarks
· Running 2 total benchmarks (1 commits * 1 environments * 2 benchmarks)
[ 0.00%] ·· Benchmarking existing-py_opt_homebrew_anaconda3_envs_ux_bin_python
[25.00%] ··· Running (mpas_ocean.ConstructFaceLatLon.time_welzl--)..
[75.00%] ··· mpas_ocean.ConstructFaceLatLon.time_welzl                                          ok
[75.00%] ··· ============ ===========
              resolution             
             ------------ -----------
                480km       156±6ms  
                120km      2.54±0.1s 
             ============ ===========

[100.00%] ··· quad_hexagon.ConstructFaceLatLon.time_welzl                                   490±3ms
```

```
asv profile mpas_ocean.ConstructFaceLatLon.time_welzl --gui=snakeviz  --python=same
Couldn't load asv.plugins._mamba_helpers because
No module named 'libmambapy'
· Discovering benchmarks
· 'mpas_ocean.ConstructFaceLatLon.time_welzl' benchmark not found
```